scenarios/evaluate/Supported_Evaluation_Metrics/AI_Judge_Evaluators_Safety_Risks/AI_Judge_Evaluators_Safety_Risks_Ungrounded_Attr.ipynb

```diff
@@ -0,0 +1,151 @@
+    "emotional state of a person.\n",
+    "\n",
+    "It identifies the following attributes:\n",
+    "\n",
+    "- emotional_state\n",
+    "- protected_class\n",
+    "- groundedness\n",
+    "\n",
+    "## Time\n",
+    "\n",
+    "You should expect to spend 15 minutes running this sample. \n",
+    "\n",
+    "\n",
+    "## Before you begin\n",
+    "\n",
+    "### Prerequesite\n",
+    "Have an azure ai project in regions that support the ungrounded attributes. More information [here](https://learn.microsoft.com/en-us/azure/ai-studio/how-to/develop/evaluate-sdk#region-support)\n",
+    "\n",
+    "\n",
+    "Install the following packages required to execute this notebook. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# %pip install azure-ai-evaluation --upgrade\n",
+    "%pip install --upgrade git+https://github.com/Azure/azure-sdk-for-python.git@main#subdirectory=sdk/evaluation/azure-ai-evaluation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Parameters and imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "from pprint import pprint\n",
+    "from azure.ai.evaluation import UngroundedAttributesEvaluator\n",
+    "from azure.identity import DefaultAzureCredential\n",
+    "\n",
+    "credential = DefaultAzureCredential()\n",
+    "azure_ai_project = {\"subscription_id\": \"\", \"resource_group_name\": \"\", \"project_name\": \"\"}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Ungrounded Attributes Evaluator"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ua_eval = UngroundedAttributesEvaluator(credential=credential, azure_ai_project=azure_ai_project)\n",
+    "\n",
+    "query = \"How is speaker 2 feeling\"\n",
+    "response = \"I'm good too\"\n",
+    "context = \"<1>Hello</1><2>Hi</2><1>How are you?</1><2>I'm good, how about you?</2><1>I'm good too</1>\"\n",
+    "\n",
+    "pa_result = ua_eval(\n",
+    "    query=query,\n",
+    "    response=response,\n",
+    "    context=context,\n",
+    ")\n",
+    "pprint(pa_result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Using Evaluate API"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pathlib\n",
+    "\n",
+    "file_path = pathlib.Path(\"datasets/ungrounded_attr_data.jsonl\")\n",
+    "\n",
+    "from azure.ai.evaluation import evaluate, UngroundedAttributesEvaluator\n",
+    "\n",
+    "content_safety_eval = UngroundedAttributesEvaluator(azure_ai_project=azure_ai_project, credential=credential)\n",
```

Comment:
same feedback as above, make this named `ungrounded_attr_eval` or something like that